src/YoutubeAtomPlaceholder.tsx

```diff
@@ -2,14 +2,17 @@ import React from 'react';
 import { css } from '@emotion/react';
 
 export const YoutubeAtomPlaceholder = ({
+    id,
     videoId,
 }: {
+    id: string;
     videoId: string;
 }): JSX.Element => {
+    const uniqueId = `youtube-placeholder-${videoId}-${id}`;
```

Comment:
Given we're creating a `uniqueId` both here and in `YoutubeAtomPlayer` does it make sense to pull it up to the `YoutubeAtom` and passing that in rather than `id`?